contracts/core/exchange-wrappers/KyberNetworkWrapper.sol

```diff
@@ -120,18 +121,16 @@ contract KyberNetworkWrapper {
      * fillQuantity                     Quantity of Set to be filled
      * attemptedfillQuantity            Quantity of Set taker attempted to fill
      *
-     * @param  _addresses               [maker, --, makerToken]
```

Comment:
Remove the rest of the parameters as well? They should refer to the struct

Reply:
Hmm? What do you mean